client-js/app/elements/labrad-app.ts

```diff
@@ -32,8 +60,12 @@ export class LabradApp extends polymer.Base {
   }
 
   doLogout(event) {
-    window.localStorage.removeItem('labrad-credentials');
-    window.sessionStorage.removeItem('labrad-credentials');
+    var key = 'labrad-credentials';
+    if (this.host) {
+      key += '.' + this.host;
+    }
+    window.localStorage.removeItem(key);
+    window.sessionStorage.removeItem(key);
```

Comment:
Yes, the credentials are stored separately for each host, since they could have different logins. Also, we use local and session storage depending on whether you say to remember the password between browser restarts.